src/nupic/regions/TPRegion.py

```diff
@@ -42,11 +42,13 @@ def _getTPClass(temporalImp):
     return TP10X2.TP10X2
   elif temporalImp == 'tm_py':
     return TP_shim.TPShim
+  elif temporalImp == 'fast_tm_py':
```

Comment:
Ah, I didn't realize it would have `py` in the name even though it is using `C++`. I've changed `fast_tm_py` to `tm_cpp`.